pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -398,7 +397,6 @@ message CommandRedeliverUnacknowledgedMessages {
 
 message CommandSuccess {
 	required uint64 request_id = 1;
-	optional Schema schema = 2;
```

Comment:
Wasn't this needed to get the consumer to be able to auto-discover the schema at the subscribe time?